assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -811,6 +811,7 @@ const baseSelectors = {
 	/**
 	 * Gets the module's screenWidgetContext.
 	 *
+	 * Returns `undefined` if dependencies are loading.
 	 * Returns `null` if there is no registered context string for the given module.
 	 * Returns `string` the registered context string, screenWidgetContext for the given module.
```

Comment:
Not sure why these are up here 🤔  can we move these down to be part of the `@return` tag instead? This is what we usually do 🙂  (e.g. `getCurrentEntityURL`)